### PR TITLE
configs/rtl8721csm: Add cxxtest config

### PR DESCRIPTION
**Pull request by sungkyu0**
This is test config for C++ build

'hello' is base config and enable cxx config

### CODE REVIEW

File: build/configs/rtl8721csm/cxxtest/Make.defs [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+  MKDEP = $(TOPDIR)/tools/mkwindeps.sh
+  ARCHINCLUDES = -I. -isystem "${shell cygpath -w $(TOPDIR)/include}" -isystem "${shell cygpath -w $(TOPDIR)/../framework/include}" -isystem "${shell cygpath -w $(TOPDIR)/../external/include}"
+  ARCHXXINCLUDES = -I. -isystem "${shell cygpath -w $(TOPDIR)/include}" -isystem "${shell cygpath -w $(TOPDIR)/../external/include}"
+  ARCHSCRIPT = -T "${shell cygpath -w $(TOPDIR)/../build/configs/$(CONFIG_ARCH_BOARD)/scripts/$(LDSCRIPT)}"
+else
+  # Linux/Cygwin-native toolchain
+  MKDEP = $(TOPDIR)/tools/mkdeps$(HOSTEXEEXT)
+  ARCHINCLUDES = -I. -isystem $(TOPDIR)/include -isystem $(TOPDIR)/../framework/include -isystem $(TOPDIR)/../external/include
+  ARCHINCLUDES += -isystem $(TOPDIR)/net/lwip/src/include
+  ARCHXXINCLUDES = -I. -isystem $(TOPDIR)/include -isystem $(TOPDIR)/../external/include
```
Comment (sunghan-chang) [inline build/configs/rtl8721csm/cxxtest/Make.defs:76]: If you have any plan to use framework on c++, you should add `-isystem $(TOPDIR)/../framework/include` as well.

File: build/configs/rtl8721csm/cxxtest/Make.defs [hunk 2]
```diff
@@ -0,0 +1,229 @@
+###########################################################################
+#
+# Copyright 2020 Samsung Electronics All Rights Reserved.
```
Comment (sunghan-chang) [inline build/configs/rtl8721csm/cxxtest/Make.defs:3]: 2021

File: build/configs/rtl8721csm/scripts/rlx8721d_img2_is.ld [hunk 3]
```diff
@@ -253,8 +253,10 @@ SECTIONS
 		__preinit_array_end = .;
 		. = ALIGN(4);
 		__init_array_start = .;
+		_sinit = ABSOLUTE(.);
```
Comment (sunghan-chang) [inline build/configs/rtl8721csm/scripts/rlx8721d_img2_is.ld:256]: Could you split commits for this ld change?
1. configs/rtl8721csm/scripts: add _sinit, _einit to support c++
2. configs/rtl8721csm: add cxxtest config

File: build/configs/rtl8721csm/cxxtest/Make.defs (hunk 2 above)
Comment (sungkyu0) [reply build/configs/rtl8721csm/cxxtest/Make.defs:3]: How about this?? Is this also need to change 2021?? 

```
#   Copyright (C) 2020 Gregory Nutt. All rights reserved.
#   Author: Gregory Nutt <gnutt@nuttx.org>
```

File: build/configs/rtl8721csm/scripts/rlx8721d_img2_is.ld (hunk 3 above)
Comment (sungkyu0) [reply build/configs/rtl8721csm/scripts/rlx8721d_img2_is.ld]: I made new PR
https://github.com/Samsung/TizenRT/pull/5383

File: build/configs/rtl8721csm/cxxtest/Make.defs (hunk 2 above)
Comment (sunghan-chang) [reply build/configs/rtl8721csm/cxxtest/Make.defs]: No. You should not modify Nuttx license itself.

**Review by sunghan-chang — APPROVED** [review]

**Pull request merged**